crates/data_components/src/duckdb.rs

```diff
@@ -65,6 +70,9 @@ pub enum Error {
     #[snafu(display("Unable to create duckdb table: {source}"))]
     UnableToCreateDuckDBTable { source: duckdb::Error },
 
+    #[snafu(display("Unable to create index on duckdb table: {source}"))]
```

Comment:
Does the source error print out the table name?

Reply:
I wasn't able to get this error to occur - but will add the table name in an upcoming PR